js/src/components/Match.vue

```diff
@@ -39,7 +41,9 @@ export default {
   data () {
     return {
       match: new Match(),
-      tournament: new Tournament()
+      tournament: new Tournament(),
+      user: this.$root.user,
+      levels: levels,
```

Comment:
Aight, then I made a semi-correct choice anyway!
Good thing it's not much to copy-paste!